packages/react-components/src/theming/FluentThemeProvider.tsx

```diff
@@ -66,11 +71,6 @@ export const FluentThemeProvider = (props: FluentThemeProviderProps): JSX.Elemen
       ChatMessage: {
         timestamp: {
           WebkitTextFillColor: fluentUITheme.palette.neutralSecondary
-        },
```

Comment:
Setting `rtl` through the Provider also gets rid of all these northstar bugs?

Reply:
Yes. I should have set 'rtl' property instead of the 'dir' property on line 83.